### PR TITLE
Send notifications on Content saving

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: Ashish Tiwari <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: server/handlers/error.go [hunk 1]
```diff
@@ -422,6 +423,9 @@ func ErrSavePattern(err error) error {
 	return errors.New(ErrSavePatternCode, errors.Alert, []string{"Error failed to save pattern"}, []string{err.Error()}, []string{"Cannot save the Pattern due to wrong path or URL"}, []string{"Check if the given path or URL of the Pattern is correct"})
 }
 
+func ErrSaveApplication(err error) error {
```
Comment (leecalcote) [inline server/handlers/error.go:426]: Did you verify that there is the proper error message for unknown components, @Revolyssup?

**Review by leecalcote — CHANGES_REQUESTED** [review]
The current error message of HTTP 400/500 / Bad request... needs to be made user-friendly.

**Comment by leecalcote** [comment]
Have error codes been enhanced and notifications sent to Notification Center for Docker Compose uploads, too, @Revolyssup?


Screen Shot 2022-11-15 at 10 16 29 PM

**Comment by Revolyssup** [comment]
> Have error codes been enhanced and notifications sent to Notification Center for Docker Compose uploads, too, @Revolyssup?
> 
> Screen Shot 2022-11-15 at 10 16 29 PM

Yes for all forms of uploads.

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**